pytest_fixtures/component/provision_pxe.py

```diff
@@ -207,16 +206,22 @@ def provisioning_host(module_ssh_key_file):
     vlan_id = settings.provisioning.vlan_id
     vm_firmware = "bios"  # TODO: Make this a fixture parameter - bios, uefi
     cd_iso = ""  # TODO: Make this an optional fixture parameter
-    with Broker(
-        workflow="deploy-configure-pxe-provisioning-host-rhv",
-        host_class=ContentHost,
-        target_vlan_id=vlan_id,
-        target_vm_firmware=vm_firmware,
-        target_vm_cd_iso=cd_iso,
-        blank=True,
-        target_memory='6GiB',
-        auth=module_ssh_key_file,
-    ) as host:
-        yield host
-        # Set host as non-blank to run teardown of the host
-        host.blank = False
+    try:
+        with Broker(
+            workflow="deploy-configure-pxe-provisioning-host-rhv",
+            host_classes={'host': ContentHost},
+            target_vlan_id=vlan_id,
+            target_vm_firmware=vm_firmware,
+            target_vm_cd_iso=cd_iso,
+            blank=True,
+            target_memory='6GiB',
```

Comment:
Do we really need 6GB? I think 2 Gb should be enough max 4 gb.

Reply:
I believe that was set by @ogajduse. Ondrej, do you think we should reduce this?

Reply:
I would leave it as is. I experienced `No space left on a disk` when I ran it with 2GiB. It is better to have more space for ramdisk, especially in discovery where we do `kexec`.
Resources should not be a concern since automation was moved out from RHVM-02. We should have plenty of space there now.